Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -14,14 +14,18 @@ public class Game {
 	}
 
 	public func playGame() {
+		let maxNumberOfGuesses = guessManager.maxNumberOfGuesses
+		view.promptNumberOfGuesses()
+		let numberOfGuesses = view.receiveNumberOfGuesses(maxNumberOfGuesses)
```

Comment:
Yes, but it shouldn't be a part of the game loop since it is more of a configuration concern.  So, you would pass it into the `GuessManager` upon its instantiation.